### ISSUE TITLE
rxjs-no-unsafe-scope not working with generics

### ISSUE THREAD

**Issue by kctang** [cartant/rxjs-tslint-rules#65]
I noticed that this rule does not work when there's generics in the function call. If I call `getProduct()` without `<Product>`, it works. I think it's a bug.

Here's a simple example:

```typescript
import { of } from 'rxjs'
import { concatMap } from 'rxjs/operators'

interface Product {
  title: string
}

class MyProductService {
  getProduct<T> () {
    return of(null)
  }

  withGeneric_notOk () {
    return of(null).pipe(
      // Unsafe scopes are forbidden
      // "rxjs-no-unsafe-scope": {
      //   "options": [
      //     {
      //       "allowDo": false,
      //       "allowMethods": true,
      //       "allowParameters": true,
      //       "allowProperties": true,
      //       "allowTap": false
      //     }
      //   ]
      // }
      concatMap(() => this.getProduct<Product>())
    )
  }

  noGeneric_ok () {
    return of(null).pipe(
      concatMap(() => this.getProduct())
    )
  }
}
```

**Comment by cartant** [comment]
I think you are right. It looks like a bug. Thanks.

**Comment by cartant** [comment]
This should be fixed in 4.13.2.

**Comment by kctang** [comment]
tested & works. thanks!
